ambry-router/src/main/java/com/github/ambry/router/AdaptiveOperationTracker.java

```diff
@@ -96,20 +96,21 @@
   @Override
   public void onResponse(ReplicaId replicaId, TrackedRequestFinalState trackedRequestFinalState) {
     super.onResponse(replicaId, trackedRequestFinalState);
-    long elapsedTime;
-    if (unexpiredRequestSendTimes.containsKey(replicaId)) {
-      elapsedTime = time.milliseconds() - unexpiredRequestSendTimes.remove(replicaId).getSecond();
-    } else {
-      elapsedTime = time.milliseconds() - expiredRequestSendTimes.remove(replicaId);
-    }
-    if (trackedRequestFinalState != TrackedRequestFinalState.TIMED_OUT
-        || !routerConfig.routerOperationTrackerExcludeTimeoutEnabled) {
-      getLatencyHistogram(replicaId).update(elapsedTime);
-      if (routerConfig.routerOperationTrackerMetricScope != OperationTrackerScope.Datacenter) {
-        // This is only used to report whole datacenter histogram for monitoring purpose
-        Histogram histogram =
-            replicaId.getDataNodeId().getDatacenterName().equals(datacenterName) ? localDcHistogram : crossDcHistogram;
-        histogram.update(elapsedTime);
+    if(trackedRequestFinalState != TrackedRequestFinalState.QUOTA_REJECTED) {
```

Comment:
no need to indent the whole code. Just do

if (trackedRequestFinalState == TrackedRequestFinalState.QUOTA_REJECTED) {
  return;
}

Reply:
done.